lib/howl/commands/edit_commands.moon

```diff
@@ -98,6 +98,25 @@ command.register
       app.editor.line_at_top = replacement.line_at_top
 
 command.register
+  name: 'buffer-sort-lines'
+  description: 'Sort the selected lines or the entire buffer'
+  handler: ->
+    editor = howl.app.editor
+    local lines
```

Comment:
Super minor nitpick...but, stylistically, why not just do:

```
local lines
lines = if editor.selection.empty
  editor.buffer.lines
else
  editor.active_lines
```

They compile to the same thing, but I think this reads a bit better.